picking_edit_printed_status/i18n/es.po

```diff
@@ -0,0 +1,15 @@
+# Translation of Odoo Server.
```

Comment:
I guess you can delete this file.

Reply:
@manuelregidor Deleted, can you review again?